src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,222 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
class fields have default values

Reply:
not fixed

Reply:
maybe I pushed to another branch, or made mistake with git. but I fixed all requested changes https://github.com/DmytroChernenko/jv-linked-list/blob/jv_linkedlist_chernenko/src/main/java/core/basesyntax/MyLinkedList.java